plugin-enclave-gradle/src/main/resources/com/r3/conclave/plugin/enclave/gradle/gramine/java.manifest.template

```diff
@@ -69,7 +71,6 @@ sgx.trusted_files = [
     "file:{{ arch_libdir }}/",
     "file:/usr/{{ arch_libdir }}/",
     "file:{{ java_home }}/",
-    "file:{{ ld_preload }}",
```

Comment:
If `ld_preload` is specified then we do need this in the trusted files list.

Reply:
Good one

Reply:
Not really sure actually why I tried to remove it, I guess because it was only meant for Python. Will readd back with the if condition.